pandas/core/indexes/datetimelike.py

```diff
@@ -121,6 +121,45 @@ def floor(self, freq):
 
     @Appender(_round_doc % "ceil")
     def ceil(self, freq):
+        """
+        Floor the index to the specified freq.
+
+        Links to the available [frequency strings](http://pandas.
+        pydata.org/pandas-docs/stable/timeseries.html#offset-aliases)
+        and [frequency objects](http://pandas.pydata.org/pandas-docs/
+        stable/timeseries.html#dateoffset-objects).
+
+        Parameters
+        ----------
+        freq : str, object
+            String or object specifying the frequency to round down to.
+
+        Returns
+        -------
+        index of same type
```

Comment:
Is it a `DatetimeInidex`? Can you add it and a short description on what it returns?